### ISSUE TITLE
`/seedoverlay 1.16.5` causes minecraft to crash.

### ISSUE THREAD

**Issue by L0um15** [xpple/SeedMapper#6]
`/seedoverlay` causes NullPointerException if seed was not set manually.

Once seed is manually set by `/seedmapper:config seed set <seed>`. It works just fine.

**Comment by L0um15** [comment]
---- Minecraft Crash Report ----
// You're mean.

Time: 07.06.21 23:23
Description: keyPressed event handler

java.lang.NullPointerException: keyPressed event handler
	at net.minecraft.client.font.TextHandler.method_27487(TextHandler.java:262)
	at net.minecraft.text.StringVisitable$3.visit(StringVisitable.java:89)
	at net.minecraft.text.StringVisitable$4.visit(StringVisitable.java:138)
	at net.minecraft.client.font.TextHandler.method_29971(TextHandler.java:261)
	at net.minecraft.client.util.ChatMessages.breakRenderedChatMessageLines(ChatMessages.java:31)
	at net.minecraft.client.gui.hud.ChatHud.addMessage(ChatHud.java:168)
	at net.minecraft.client.gui.hud.ChatHud.addMessage(ChatHud.java:158)
	at net.minecraft.client.gui.hud.ChatHud.addMessage(ChatHud.java:154)
	at dev.xpple.seedmapper.util.chat.Chat.toChat(Chat.java:12)
	at dev.xpple.seedmapper.util.chat.Chat.error(Chat.java:45)
	at dev.xpple.seedmapper.util.chat.Chat.error(Chat.java:50)
	at dev.xpple.seedmapper.command.ClientCommandManager.executeCommand(ClientCommandManager.java:71)
	at net.minecraft.client.network.ClientPlayerEntity.handler$zzd000$sendChatMessage(ClientPlayerEntity.java:1098)
	at net.minecraft.client.network.ClientPlayerEntity.sendChatMessage(ClientPlayerEntity.java)
	at net.minecraft.client.gui.screen.Screen.sendMessage(Screen.java:314)
	at net.minecraft.client.gui.screen.Screen.sendMessage(Screen.java:306)
	at net.minecraft.client.gui.screen.ChatScreen.keyPressed(ChatScreen.java:95)
	at net.minecraft.client.Keyboard.method_1454(Keyboard.java:298)
	at net.minecraft.client.gui.screen.Screen.wrapScreenError(Screen.java:448)
	at net.minecraft.client.Keyboard.onKey(Keyboard.java:292)
	at net.minecraft.client.Keyboard.method_22678(Keyboard.java:385)
	at net.minecraft.util.thread.ThreadExecutor.execute(ThreadExecutor.java:86)
	at net.minecraft.client.Keyboard.method_22676(Keyboard.java:385)
	at org.lwjgl.glfw.GLFWKeyCallbackI.callback(GLFWKeyCallbackI.java:37)
	at org.lwjgl.system.JNI.invokeV(Native Method)
	at org.lwjgl.glfw.GLFW.glfwPollEvents(GLFW.java:3101)
	at com.mojang.blaze3d.systems.RenderSystem.flipFrame(RenderSystem.java:93)
	at net.minecraft.client.util.Window.swapBuffers(Window.java:339)
	at net.minecraft.client.MinecraftClient.render(MinecraftClient.java:1026)
	at net.minecraft.client.MinecraftClient.run(MinecraftClient.java:624)
	at net.minecraft.client.main.Main.main(Main.java:187)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:226)
	at net.fabricmc.loader.launch.knot.Knot.launch(Knot.java:146)
	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:28)
	at net.fabricmc.devlaunchinjector.Main.main(Main.java:86)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at net.minecraft.client.font.TextHandler.method_27487(TextHandler.java:262)
	at net.minecraft.text.StringVisitable$3.visit(StringVisitable.java:89)
	at net.minecraft.text.StringVisitable$4.visit(StringVisitable.java:138)
	at net.minecraft.client.font.TextHandler.method_29971(TextHandler.java:261)
	at net.minecraft.client.util.ChatMessages.breakRenderedChatMessageLines(ChatMessages.java:31)
	at net.minecraft.client.gui.hud.ChatHud.addMessage(ChatHud.java:168)
	at net.minecraft.client.gui.hud.ChatHud.addMessage(ChatHud.java:158)
	at net.minecraft.client.gui.hud.ChatHud.addMessage(ChatHud.java:154)
	at dev.xpple.seedmapper.util.chat.Chat.toChat(Chat.java:12)
	at dev.xpple.seedmapper.util.chat.Chat.error(Chat.java:45)
	at dev.xpple.seedmapper.util.chat.Chat.error(Chat.java:50)
	at dev.xpple.seedmapper.command.ClientCommandManager.executeCommand(ClientCommandManager.java:71)
	at net.minecraft.client.network.ClientPlayerEntity.handler$zzd000$sendChatMessage(ClientPlayerEntity.java:1098)
	at net.minecraft.client.network.ClientPlayerEntity.sendChatMessage(ClientPlayerEntity.java)
	at net.minecraft.client.gui.screen.Screen.sendMessage(Screen.java:314)
	at net.minecraft.client.gui.screen.Screen.sendMessage(Screen.java:306)
	at net.minecraft.client.gui.screen.ChatScreen.keyPressed(ChatScreen.java:95)
	at net.minecraft.client.Keyboard.method_1454(Keyboard.java:298)

-- Affected screen --
Details:
	Screen name: net.minecraft.client.gui.screen.ChatScreen
Stacktrace:
	at net.minecraft.client.gui.screen.Screen.wrapScreenError(Screen.java:448)
	at net.minecraft.client.Keyboard.onKey(Keyboard.java:292)
	at net.minecraft.client.Keyboard.method_22678(Keyboard.java:385)
	at net.minecraft.util.thread.ThreadExecutor.execute(ThreadExecutor.java:86)
	at net.minecraft.client.Keyboard.method_22676(Keyboard.java:385)
	at org.lwjgl.glfw.GLFWKeyCallbackI.callback(GLFWKeyCallbackI.java:37)
	at org.lwjgl.system.JNI.invokeV(Native Method)
	at org.lwjgl.glfw.GLFW.glfwPollEvents(GLFW.java:3101)
	at com.mojang.blaze3d.systems.RenderSystem.flipFrame(RenderSystem.java:93)

-- Affected level --
Details:
	All players: 1 total; [ClientPlayerEntity['Player123'/354, l='ClientLevel', x=-240.50, y=63.00, z=-189.50]]
	Chunk stats: Client Chunk Cache: 841, 529
	Level dimension: minecraft:overworld
	Level spawn location: World: (-240,64,-192), Chunk: (at 0,4,0 in -15,-12; contains blocks -240,0,-192 to -225,255,-177), Region: (-1,-1; contains chunks -32,-32 to -1,-1, blocks -512,0,-512 to -1,255,-1)
	Level time: 285 game time, 285 day time
	Server brand: fabric
	Server type: Integrated singleplayer server
Stacktrace:
	at net.minecraft.client.world.ClientWorld.addDetailsToCrashReport(ClientWorld.java:454)
	at net.minecraft.client.MinecraftClient.addDetailsToCrashReport(MinecraftClient.java:2166)
	at net.minecraft.client.MinecraftClient.run(MinecraftClient.java:640)
	at net.minecraft.client.main.Main.main(Main.java:187)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:226)
	at net.fabricmc.loader.launch.knot.Knot.launch(Knot.java:146)
	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:28)
	at net.fabricmc.devlaunchinjector.Main.main(Main.java:86)

**Comment by xpple** [comment]
Hey! First of all thank you for checking out my mod!

Although the client shouldn't crash, it is supposed functionality to set the seed first using the `/config` command, and then afterwards execute the overlay command. This will be fixed along with the final 1.16 release.

**Comment by xpple** [comment]
What did you `config.json` file look like when you executed the command? Did `seed` map to `null`?

**Comment by L0um15** [comment]
Config.json
```json
{
  "automate": {
    "enabled": false
  },
  "seed": null,
  "ignoredBlocks": [

  ]
}
```

**Comment by L0um15** [comment]
Issue is somewhere here i think
```java
JsonElement element = Config.get("seed");
if (element instanceof JsonNull) {
      throw NULL_POINTER_EXCEPTION.create("seed");
}
```

**Comment by L0um15** [comment]
Seedmapper inside IntellijIdea works just fine
![image](https://user-images.githubusercontent.com/44427021/121310846-15ef7980-c904-11eb-968d-0dcac6df2098.png)

**Comment by L0um15** [comment]
It turned out, that b1.0 was published with `Initial Commit` and was fixed in `e392985`. I should've pay more attention.

**Comment by xpple** [comment]
Ah of course, my bad. I’ve committed some changes that aren’t in the release yet. The final 1.16 release will be stable.
